_config/debugbar.yml

```diff
@@ -51,3 +51,5 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\Control\Controller:
   extensions:
     - LeKoala\DebugBar\Extension\ControllerExtension
+LeKoala\DebugBar\DebugBar:
+  show_namespaces: false
```

Comment:
If you bound it to the database collector class it'd be more obvious what it's relating to